docs/references/fluxyaml-config-files.md

```diff
@@ -1,4 +1,4 @@
-# Manifest generation through `.flux.yaml` configuration files
+# Manifest generation through .flux.yaml configuration files
```

Comment:
Does using `` in the titles cause issues with something?

Reply:
It was broken when I first looked at things. Looks like this is fixed now - maybe with one of the mkdocs plugins we started using now? :man_shrugging: I'll revert the ` removals.

Reply:
Mhmh, maybe the `pymdownx.superfences` plugin.